### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by abdulrashid123**
please accept commit

### CODE REVIEW

File: src/features/counter/Counter.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
+import React, { useState } from 'react';
```
Comment (Kunal3737) [inline src/features/counter/Counter.js:1]: We don't need this file as of now this was done for testing purpose, so now as we are merging to the beta branch, this counter files should be removed.

File: README.md [hunk 2]
```diff
@@ -1 +1,44 @@
...
+
+Instead, it will copy all the configuration files and the transitive dependencies (Webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you’re on your own.
+
+You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
+
+## Learn More
+
+You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+
+To learn React, check out the [React documentation](https://reactjs.org/).
```
Comment (gaurav7557) [inline README.md:44]: Add the readme that we had created for the same

File: src/features/login/loginSlice.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    try {
+        //const response = await fetch(`url`); //where you want to fetch data
+        //Your Axios code part.
+        const response = await axios.post('/api-token-auth/',data)
+        ls.set('token', response.data.token)
+        return response.data.token
+  
+    } 
+      catch (error) {
+          alert("Authentication denied")
```
Comment (Kunal3737) [inline src/features/login/loginSlice.js:24]: Use react-toastify instead of showing general alerts.

File: src/app/store.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
+import { configureStore } from '@reduxjs/toolkit';
+import counterReducer from '../features/counter/counterSlice';
+import loginReducer from '../features/login/loginSlice'
+import shopReducer from '../features/shop/shopSlice'
+export const store = configureStore({
+  reducer: {
+    counter: counterReducer,
```
Comment (gaurav7557) [inline src/app/store.js:7]: Nit : Indentation

File: src/features/counter/Counter.js [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+  incrementAsync,
+  incrementIfOdd,
+  selectCount,
+} from './counterSlice';
+import styles from './Counter.module.css';
+
+export function Counter() {
+  const count = useSelector(selectCount);
+  const dispatch = useDispatch();
+  const [incrementAmount, setIncrementAmount] = useState('2');
```
Comment (gaurav7557) [inline src/features/counter/Counter.js:16]: constants can be defined separately in a file, you can incorporate all the string related constants. It helps in better understanding.

File: src/features/counter/Counter.module.css [hunk 6]
```diff
@@ -0,0 +1,78 @@
+.row {
```
Comment (gaurav7557) [inline src/features/counter/Counter.module.css:1]: Use relative dimensions like % and move the colors to a variable in the top of the file, @Kunal3737

File: src/features/counter/counterAPI.js [hunk 7]
```diff
@@ -0,0 +1,6 @@
+// A mock function to mimic making an async request for data
+export function fetchCount(amount = 1) {
+  return new Promise((resolve) =>
+    setTimeout(() => resolve({ data: amount }), 500)
```
Comment (gaurav7557) [inline src/features/counter/counterAPI.js:4]: 500 can be marked as a constant

File: src/screens/Employee/Employee.css [hunk 8]
```diff
@@ -0,0 +1,52 @@
+.employee__main  {
+    /* Remember to use the other versions for IE 10 and older browsers! */
```
Comment (gaurav7557) [inline src/screens/Employee/Employee.css:2]: @Kunal3737 Cover this CSS as well

File: src/screens/HomeScreen/HomeScreen.js [hunk 9]
```diff
@@ -0,0 +1,23 @@
...
+    const dispatch = useDispatch()
+    const shops = useSelector(state => state.shop.shops)
+    return (
+        <div className="home__container">
+            {shops?.map(item => (
+                <Card onClick={() => dispatch(currentShop(item.id))} style={{ width: '40rem',marginTop:"2rem"}}>
+                <Card.Body style={{textAlign:'center',fontSize:'2rem',margin:'1rem'}}>{item.name}</Card.Body>
+              </Card>
+
+            ))}
```
Comment (gaurav7557) [inline src/screens/HomeScreen/HomeScreen.js:17]: maintain indentation for better clarity

File: src/features/login/loginSlice.js (hunk 3 above)
Comment (gaurav7557) [reply src/features/login/loginSlice.js:24]: Kunal will add

**Pull request merged**